### PR TITLE
Find mygui

### PR DESCRIPTION
**Pull request by kpp**
`FindMyGUI.cmake` set MYGUI_* variables, which is incorrect acording to CMake docs. I fixed that.

### CODE REVIEW

File: CI/before_script.msvc.sh [hunk 1]
```diff
@@ -428,11 +428,7 @@ printf "MyGUI 3.2.2... "
 		mv MyGUI-3.2.2-win$BITS MyGUI
 	fi
 
-	MYGUI_SDK="`real_pwd`/MyGUI"
-
-	add_cmake_opts -DMYGUISDK="$MYGUI_SDK" \
-		-DMYGUI_INCLUDE_DIRS="$MYGUI_SDK/include/MYGUI" \
-		-DMYGUI_PREQUEST_FILE="$MYGUI_SDK/include/MYGUI/MyGUI_Prerequest.h"
+	export MYGUI_HOME="`real_pwd`/MyGUI"
```
Comment (kpp) [inline CI/before_script.msvc.sh]: Use MYGUI_HOME as it is said in wiki

File: cmake/FindFreetype.cmake [hunk 2]
```diff
@@ -78,21 +77,13 @@ find_path(Freetype_OLD_INCLUDE_DIR
     PATH_SUFFIXES freetype2
     NO_DEFAULT_PATH
 )
-
-# get version from freetype.h
-find_file(Freetype_HEADER
-    NAMES freetype.h
-    PATH_SUFFIXES freetype
-    PATHS ${Freetype_OLD_INCLUDE_DIR}
+libfind_version_n_header(Freetype
```
Comment (kpp) [inline cmake/FindFreetype.cmake]: Rewrite `find_file` and `get_version_from_n_defines` with `libfind_version_n_header`

**Comment by psi29a** [comment]
@scrawl or @ace13 

Can one of you give me access to appveyor so I can kick it like I do travis-ci to rebuild the last build?

@kpp this reminds me that I need to decouple libmygui's opengl platform away from ogre so you don't have to install the whole thing.

**Comment by ace13** [comment]
Might be a good idea to link in organisation access on AppVeyor, should let people do such things through their github accounts.
Don't remember how to do such a thing though, and @scrawl is the one with the AppVeyor project either way.

**Comment by scrawl** [comment]
> Can one of you give me access to appveyor so I can kick it like I do travis-ci to rebuild the last build?

I just ticked some boxes, do you see the rebuild button now?

**Comment by psi29a** [comment]
Sorry, still don't see any rebuild buttons.  Am I on a team (your team) ?

the webhook is linked against the scrawl account.

**Comment by scrawl** [comment]
The button should be here:
rebuild_1

This is what I changed on appveyor:

rebuild_2

rebuild_3

**Comment by psi29a** [comment]
I don't have RE-BUILD PR, just LOG.

**Comment by psi29a** [comment]
... found it. Thanks scrawl.

For future generations to come, here is the deal. If you are currently logged into your account on appveyor, you won't have access to someone else's project. You have to log out, then log back in. It will ask you under which account to log into.  In this instance instead of choosing my psi29a account, I had to choose scrawl's account. Then I had access to OpenMW and the rebuild button.

I have to say, this is most unhandy way of handling accounts I've ever seen. I'm sure there is a technical reason for it, but it still sucks.  Anyway... moving.

**Comment by ace13** [comment]
Which is why I suggested setting up the AppVeyor project under an organization account, that way you don't have to do per-user trickery.

Probably a bitch to set up though, considering how "easily" the rest of AppVeyor works.

**Comment by kpp** [comment]
> @kpp this reminds me that I need to decouple libmygui's opengl platform away from ogre so you don't have to install the whole thing.

I didn't get it...

**Comment by Allofich** [comment]
Excuse me for intruding, but I think maybe there was a kind of communication breakdown here that brought this discussion to a halt? @psi29a, maybe you can re-explain to kpp?

**Comment by psi29a** [comment]
Something Debian related, nothing to do with what @kpp is doing.

**Comment by Allofich** [comment]
OK, I thought maybe his "I didn't get it.." meant he didn't understand what you had said, and the discussion seemed to have ended there, hence my post. :)

**Comment by kpp** [comment]
Well, I didn't understand what psi29a had said.

**Comment by Allofich** [comment]
He says it's unrelated, so I guess you don't need to worry about it for this PR.
kpp, I thought that maybe you were waiting for an answer from him. Never mind if I'm mistaken.

**Comment by kpp** [comment]
@scrawl you are free to merge it.

**Pull request merged**